security-admin/src/main/webapp/scripts/views/common/ProfileBar.js

```diff
@@ -77,6 +77,32 @@ define(function(require){
 				
 			});
 		},
+		onLogoutSSO : function(checksso) {
+			var url = 'security-admin-web/logout.html',
+			that = this;
+			$.ajax({
+				url : url,
+				type : 'GET',
+				headers : {
+					"cache-control" : "no-cache"
+				},
+				success : function() {
+					if (checksso.altiSSO.isEnabled == 'true') {
+						window.location.replace(checksso.altiSSO.logoutURL);
```

Comment:
I still don't see where you invalidate session cookie?

Reply:
forgot to answer that. ranger uses the spring security framework and the session cookies are deleted on logout by the framework. the configuration is provided at https://github.com/apache/ranger/blob/c4020f89b929f323d133fbe81d2571deca29f934/security-admin/src/main/resources/conf.dist/security-applicationContext.xml#L58.